DEVELOPERS

```diff
@@ -215,3 +216,28 @@ If in doubt, reach out on IRC or via a "question" Issue on GitHub.
   - no scraping HTML!
 - New code should be covered by unit-tests
 
+
+Creating a Release
+==================
+
+- Install the tools required:
```

Comment:
I would suggest doing this in a virtualenv (and telling readers of this to use a virtualenv as well)